app/views/admin/tickets/index.html.haml

```diff
@@ -28,6 +29,11 @@
               %td
                 = humanized_money_with_symbol ticket.tickets_turnover
               %td
+                - if ticket.registration_ticket?
```

Comment:
maybe writing this shorter: :thinking: 

= ticket.registration_ticket? ? 'Yes' : 'No'